mysite/models/models.py

```diff
@@ -1454,6 +1469,19 @@ def _parse_w4_lab_bonuses(self, raw_lab):
                 "Enabled": True,
                 "Value": 1
             }
+            
+    def _parse_w4_jewels(self, raw_lab):
+        #TODO: Account for if the jewel is actually connected.
+        jewelmulti = 1+0.5+0.1*raw_lab[14][16] # Will have to be fixed when labbonuses are implemented properly. index 16 is purple navette
```

Comment:
* Move this `jewelmulti` variable down to the calculate function. 
* Change to checking `Enabled` so it will be properly accounted for in the future.
`jewelmulti = 1 + ((.5 * spelunker enabled) + (0.1 * pure opal enabled))`